Source/WebInspectorUI/UserInterface/Views/GraphicsOverviewContentView.css

```diff
@@ -27,7 +27,6 @@
     background-color: var(--background-color-content);
```

Comment:
actually i think this is the one we want to remove since this is the one that's overridden (i.e. if we remove the other one then this will take effect and it'll result in a different look)